ZelBack/src/services/fluxService.js

```diff
@@ -639,47 +837,57 @@ async function getFluxInfo(req, res) {
   }
 }
 
-async function adjustCruxID(req, res) {
-  try {
-    const authorized = await verificationHelper.verifyPrivilege('admin', req);
-    if (authorized === true) {
-      let { cruxid } = req.params;
-      cruxid = cruxid || req.query.cruxid;
-      if (!cruxid) {
-        throw new Error('No Crux ID provided');
-      }
-      if (!cruxid.includes('@')) {
-        throw new Error('Invalid Crux ID provided');
-      }
-      if (!cruxid.includes('.crux')) {
-        throw new Error('Invalid Crux ID provided');
-      }
-      const fluxDirPath = path.join(__dirname, '../../../config/userconfig.js');
-      const dataToWrite = `module.exports = {
-  initial: {
-    ipaddress: '${userconfig.initial.ipaddress || '127.0.0.1'}',
-    zelid: '${userconfig.initial.zelid || config.fluxTeamZelId}',
-    kadena: '${userconfig.initial.kadena || ''}',
-    testnet: ${userconfig.initial.testnet || false},
-    apiport: ${Number(userconfig.initial.apiport || config.apiport)},
-  }
-}`;
-
-      await fsPromises.writeFile(fluxDirPath, dataToWrite);
-
-      const successMessage = messageHelper.createSuccessMessage('CruxID adjusted');
-      res.json(successMessage);
-    } else {
-      const errMessage = messageHelper.errUnauthorizedMessage();
-      res.json(errMessage);
-    }
-  } catch (error) {
-    log.error(error);
-    const errMessage = messageHelper.createErrorMessage(error.message, error.name, error.code);
-    res.json(errMessage);
-  }
-}
-
+// /**
+//  * To adjust the current CruxID (now deprecated) that is being used with FluxOS. Only accessible by admins.
+//  * @param {object} req Request.
+//  * @param {object} res Response.
+//  */
+// async function adjustCruxID(req, res) {
```

Comment:
Still keep for now as function is referenced in api. Soon will be most likely changed to FIO (as rename)